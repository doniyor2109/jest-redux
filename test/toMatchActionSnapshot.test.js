import { pureActions } from './stub';

describe('toHaveInitialState', () => {
  it('should have correct initial state', () => {
    expect(pureActions).toMatchActionSnapshot();
  });
});
