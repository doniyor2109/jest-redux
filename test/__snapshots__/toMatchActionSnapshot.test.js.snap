// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toHaveInitialState should have correct initial state: add should match with snapshot 1`] = `
Object {
  "payload": 1,
  "type": "ADD",
}
`;

exports[`toHaveInitialState should have correct initial state: remove should match with snapshot 1`] = `
Object {
  "payload": 1,
  "type": "REMOVE",
}
`;
